engine/runtime/src/main/java/org/enso/interpreter/runtime/data/Type.java

```diff
@@ -170,4 +227,16 @@ boolean isNull(@CachedLibrary("this") InteropLibrary self) {
   public String toString() {
     return toDisplayString(true);
   }
+
+  public Type getEigentype() {
```

Comment:
There already is `Type getType()`...

Reply:
Yes, but that is for library usage, and given this is already a type, and there's also it's supertype, I want to be super clear with the getters.